pkg/config/config.go

```diff
@@ -6,8 +6,8 @@ import (
 	"path"
 )
 
-// K3sVersion default version
```

Comment:
By now default pinning a branch we could break peoples automation when the upstream main changes. The current version pin saves us from any errors whena new version is released. What if k3s next version didn't work on some devices it didn't work on before? 

I could have missed the point, but it looks like the above is the case.

Reply:
Hey @Waterdrips i understood your comment and will be fine with changing that behavior. From my point of view it was a more obvious behavior to just install the latest stable version, because that is what the user normally expect.

I will change this later today.